articles/github/github-actions.md

```diff
@@ -12,9 +12,9 @@ ms.custom: github-actions-azure
 
 # What is GitHub Actions for Azure
 
-[GitHub Actions](https://help.github.com/articles/about-github-actions) helps you automate your software development workflows from within GitHub. You can deploy workflows in the same place where you store code and collaborate on pull requests and issues.
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions) helps you automate your software development workflows from within GitHub. You can deploy workflows in the same place where you store code and collaborate on pull requests and issues.
```

Comment:
I changed this to go to the Actions landing page instead of to the "Learn GitHub Actions" article. If you prefer to link to the "Learn GitHub Actions" article instead, I recommend updating `https://help.github.com/articles/about-github-actions` to `https://docs.github.com/en/free-pro-team@latest/actions/learn-github-actions`.